src/common/components/Icon/index.js

```diff
@@ -12,10 +12,17 @@ type IconProps = {
   src?: string,
   button?: boolean,
   children?: any,
+  applyCount?: number,
 };
 
-const Icon = ({ name, size, className, src, button, children, ...props }: IconProps) => {
+const Icon = ({ name, size, className, src, button, children, applyCount, ...props }: IconProps) => {
+  const APPLY_COUNT_THRESHOLD = 1;
   let image: string;
+  let applyBadge: any;
+
+  if (applyCount && applyCount > APPLY_COUNT_THRESHOLD) {
```

Comment:
agreed - icon is used for more than just gw2 icons though, its used for any images really.

maybe we can make a Gw2Icon that wraps Icon? at least it'd be more focused. im happy to ignore it for now and do it later though.